cmd/dep/testdata/gom/Gomfile

```diff
@@ -0,0 +1,2 @@
+gom "github.com/sdboyer/deptest", :commit => "3f4c3bea144e112a69bbe5d8d01c1b09a544253f"
```

Comment:
It is important that tests for `:branch` and `:tag` are added, in addition to an integration test.